src/main/java/it/pagopa/pn/bff/service/InfoPaService.java

```diff
@@ -58,4 +57,26 @@ public Flux<BffInstitutionProduct> getInstitutionProducts(String xPagopaPnUid, C
                 .map(product -> ProductMapper.modelMapper.toBffInstitutionProduct(product, pnBffConfigs, xPagopaPnCxId))
                 .onErrorMap(WebClientResponseException.class, pnBffExceptionUtility::wrapException);
     }
+
+    /**
+     * Get the list of groups for the user
+     *
+     * @param xPagopaPnUid      User Identifier
+     * @param xPagopaPnCxId     Public Administration id
+     * @param xPagopaPnCxGroups  Public Administration Group id List
+     * @param statusFilter    Filter for the status of the groups
+     * @return the list of groups
+     */
+    public Flux<PaGroup> getGroups(String xPagopaPnUid, String xPagopaPnCxId, List<String> xPagopaPnCxGroups, BffPaGroupStatus statusFilter){
+        log.info("getGroups");
+
+        Flux<it.pagopa.pn.bff.generated.openapi.msclient.external_registries_selfcare.model.PaGroup> paGroups = pnExternalRegistriesClient.getGroups(
+                xPagopaPnUid,
+                xPagopaPnCxId,
+                xPagopaPnCxGroups,
+                null
```

Comment:
Pay attention to this. We need to pass the statusFilter to the ext-registry api.